src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.macios.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.ObjectModel;
+using Foundation;
+using Microsoft.Maui.Graphics.Platform;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+public partial class MauiDrawingView : PlatformTouchGraphicsView
+{
+	readonly List<UIScrollView> scrollViewParents = new();
+	
+	/// <inheritdoc />
+	public override void TouchesBegan(NSSet touches, UIEvent? evt)
+	{
+		base.TouchesBegan(touches, evt);
+		DetectScrollViews();
+		SetParentTouches(false);
+
+		var touch = (UITouch)touches.AnyObject;
+		OnStart(touch.PreviousLocationInView(this).AsPointF());
+	}
+
+	/// <inheritdoc />
+	public override void TouchesMoved(NSSet touches, UIEvent? evt)
+	{
+		base.TouchesMoved(touches, evt);
+		var touch = (UITouch)touches.AnyObject;
+		var currentPoint = touch.LocationInView(this);
+		OnMoving(currentPoint.AsPointF());
+	}
+
+	/// <inheritdoc />
+	public override void TouchesEnded(NSSet touches, UIEvent? evt)
+	{
+		base.TouchesEnded(touches, evt);
+		OnFinish();
+		SetParentTouches(true);
+	}
+
+	/// <inheritdoc />
+	public override void TouchesCancelled(NSSet touches, UIEvent? evt)
+	{
+		base.TouchesCancelled(touches, evt);
+		OnCancel();
+		SetParentTouches(true);
+	}
+
+	void DetectScrollViews()
+	{
+		if (scrollViewParents.Count > 0)
+		{
+			return;
+		}
+
+		var parent = Superview;
+
+		while (parent is not null)
+		{
+			if (parent.GetType() == typeof(UIScrollView))
+			{
+				scrollViewParents.Add((UIScrollView)parent);
+			}
+
```

Comment:
can't we use patter-match?
```suggestion
                        if (parent is UIScrollView scrollView)
			{
				scrollViewParents.Add(scrollView);
			}

```